setup.py

```diff
@@ -195,6 +195,7 @@ def cython_extensions_and_necessity():
   extensions = [
       _extension.Extension(
           name=module_name,
+          language='c++',
```

Comment:
I'm okay with this quick check you've done, thanks. The RC will hopefully tell us more if we've actually broken somebody, but I feel more confident now.